### ISSUE TITLE
windows: Update processes table to contain s{u,g}id, and e{u,g}id values

### ISSUE THREAD

**Issue by muffins** [osquery/osquery#2791]
Now that the Users table is populating the `uid` and `gid` values, let's pull these into the processes table.

We'll need to pull in the `euid` and `suid` values for each process. This data should be attainable from polling the current running Token.

**Comment by theopolis** [comment]
Seems like UID and GIDs are reported, now for SUID/SGID/EUID/EGID. :)

**Comment by mike-myers-tob** [comment]
> Seems like UID and GIDs are reported, now for SUID/SGID/EUID/EGID. :)

I think maybe we've gone as far as we can go with this table schema, because the Windows security model doesn't have the concept of the "effective user ID" at the process level, the way UNIX does.

Instead, it has "impersonation" at the thread-level. If a thread has an impersonation token in addition to its primary access token (the one being parsed for the `processes` table in order to get a SID, from which in turn we are deriving a `UID` and `GID` equivalent).

TL;DR I would recommend we close this issue.
